src/shapes/Circle.ts

```diff
@@ -205,6 +210,10 @@ export class Circle extends FabricObject implements CircleProps {
   }
 
   /* _FROM_SVG_END_ */
+
+  static fromObject(object: SerializedCircleProps): Promise<Circle> {
+    return super.fromObject(object) as Promise<Circle>;
+  }
```

Comment:
something i noticed in the previous PR too, i would argue this can be solved with a declaration?
is weird that TS would force me to add code that does nothing to obey types.

Reply:
correct
I was lazy

Reply:
This is another TS problem. Static class methods can't be generic using straight forward TS